powersOfTwo.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
```

Comment:
You forgot the 2^0 = 1 case! Aside from that, nice job. Also, again, don't forget your comments!